include/JSystem/JKernel/JKRFile.h

```diff
@@ -0,0 +1,103 @@
+#ifndef JKRFILE_H
+#define JKRFILE_H
+
+#include "types.h"
+#include "dolphin/dvd.h"
+#include "dolphin/os.h"
+#include "JSystem/JSupport/JSUList.h"
+#include "JSystem/JSupport/JSUStream.h"
+#include "JSystem/JKernel/JKRAram.h"
+#include "JSystem/JKernel/JKRFile.h"
+#include "JSystem/JKernel/JKRDisposer.h"
+
+#define JKR_ISALIGNED(addr, alignment) ((((u32)addr) & (((u32)alignment)-1)) == 0)
+
+struct JKRDvdFileInfo : public DVDFileInfo {
```

Comment:
Shouldn't this be after the class? 
(This gets a declaration syntax error when compiling)